azure-pipelines.yml

```diff
@@ -97,6 +98,9 @@ stages:
   jobs:
   - template: .azure-pipelines/build-docker-sonic-vs-template.yml
     parameters:
+      swss_common_branch: master
```

Comment:
When checked out new release branch. For example 202311.
The pipeline maybe break for dependency issue.

Reply:
'$(BUILD_BRANCH)' is safe for new release branches.